goalgorilla/open_social
# Issue #3170795 by rolki: Hide entities from custom content list block if user do not have access

## Problem
Users that don’t have access to entities can see these entities by custom content list block. Behavior should be the same as in drupal core (if do not access then hide the entity include teasers)

## Solution
Add access check condition

## Issue tracker
https://www.drupal.org/project/social/issues/3170795

## How to test
- Create a custom content list block /block/add/custom_content_list
- Select some entity type to which for example authenticated user does not have access
- Go to the page with this block under authenticated user
- You can see entities

## Screenshots
![image](https://user-images.githubusercontent.com/50984627/93072174-b83a4380-f689-11ea-8043-3a22d681b773.png)
